calico-enterprise/getting-started/bare-metal/about.mdx

```diff
@@ -40,94 +40,108 @@ To learn how to restrict traffic to/from hosts and VMs using Calico network poli
 
 ## How to
 
-### Set up your Kubernetes cluster to work with a non-cluster host or VM
-
-1. Apply the `NonClusterHost` custom resource. The `endpoint` field is required. It defines the remote endpoint for the non-cluster host log forwarder.
-
-   ```bash
-   kubectl create -f - <<EOF
-   apiVersion: operator.tigera.io/v1
-   kind: NonClusterHost
-   metadata:
-     name: tigera-secure
-   spec:
-     endpoint: <endpoint for non-cluster host log forwarder>
-   EOF
-   ```
-
-   Wait until the manager shows a status of Available, then proceed to the next step.
-
-1. Obtain the token for `tigera-noncluster-host` service account.
-
-   ```bash
-   kubectl get secret -n calico-system tigera-noncluster-host -o jsonpath='{.data.token}' | base64 --decode
-   ```
-
-1. Use a text editor to create a kubeconfig file for non-cluster host or VM.
-
-   ```yaml
-   apiVersion: v1
-   kind: Config
-   current-context: noncluster-hosts
-   preferences: {}
-   clusters:
-   - cluster:
-       certificate-authority-data: <your cluster certificate>
-       server: <your cluster server>
-     name: noncluster-hosts
-   contexts:
-   - context:
-       cluster: noncluster-hosts
-       user: tigera-noncluster-host
-     name: noncluster-hosts
-   users:
-   - name: tigera-noncluster-host
-     user:
-       token: <token from previous step>
-   ```
-
-   Take the cluster information from an already existing kubeconfig.
-
-1. Create a [`HostEndpoint` resource](../../reference/host-endpoints/overview.mdx) for each non-cluster host or VM. The `node` and `expectedIPs` fields are required to match the hostname and the expected interface IP addresses.
-
-1. Follow [Configure access to the $[prodname] web console](../../operations/cnx/access-the-manager.mdx) page to configure the $[prodname] web console access.
+### Set up your Kubernetes cluster to work with Accepted Valuesa non-cluster host or VM
+
+1. Apply the `NonClusterHost` custom resource. This resource enables the cluster-side ingestion endpoint to receive logs from non-cluster hosts. Additionally, it provides a dedicated Typha deployment for managing communication between the non-cluster host agent and Typha.
+
+    To ensure a secure connection, non-cluster host agents will establish an mTLS connection via the `typhaEndpoint` endpoint. As a result, the external ingress or load balancer must be configured to pass through TCP Layer 4 traffic.
```

Comment:
If you think people need to know details about the connection, then consider moving that out of the step procedure to an introduction that explains what's going on. Otherwise, you can leave this out.

I'd go with something like this:

```
1. Create an apply a `NonClusterHost` custom resource.
   This resource enables the cluster-side ingestion endpoint to receive logs from non-cluster hosts.
    Additionally, it provides a dedicated Typha deployment for managing communication between the non-cluster host agent and Typha.
```

Reply:
Updated as suggested. The details for `typhaEndpoint` endpoint is moved to the "Configure hosts to communicate with your Kubernetes cluster" section below. The layer 4 pass through requirement is a bit untypical to configure an ingress or LB so I want to mention that in our doc.